components/bigdatacorp/actions/get-address-data/get-address-data.mjs

```diff
@@ -0,0 +1,40 @@
+import app from "../../bigdatacorp.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "bigdatacorp-get-address-data",
+  name: "Get Address Data",
+  description: "Returns the available data for a CEP number according to the selected dataset. [See the documentation](https://docs.bigdatacorp.com.br/plataforma/reference/enderecos_legal_amazon)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    doc: {
+      propDefinition: [
+        app,
+        "doc",
+      ],
+    },

```

Comment:
Shouldn't this be `zipcode` instead of `doc`?
```suggestion
    zipcode: {
      type: "string",
      label: "Zipcode",
      description: "The postal code to search for",
    },

```